packages/ckeditor5-dev-env/lib/index.js

```diff
@@ -26,47 +32,114 @@ const tasks = {
 			packages: options.packages
 		};
 
-		const functionToExecute = ( repositoryName, repositoryPath ) => {
+		const generateChangelogForSinglePackage = ( repositoryName, repositoryPath ) => {
 			process.chdir( repositoryPath );
 
 			return tasks.generateChangelog();
 		};
 
-		return executeOnDependencies( execOptions, functionToExecute )
+		return executeOnDependencies( execOptions, generateChangelogForSinglePackage )
 			.then( () => {
 				process.chdir( options.cwd );
 			} );
 	},
 
 	/**
-	 * Generates the changelog for dependencies.
+	 * Task releases the dependencies. It collects packages that will be release,
+	 * validates whether the packages can be released and gather required data from the user.
 	 *
 	 * @param {Object} options
-	 * @params {String} options.cwd Current work directory.
-	 * @params {String} options.packages A relative path to the packages.
-	 * @params {String} options.token GitHub token used to authenticate.
-	 * @params {Object} options.dependencies Dependencies with versions of other CKEditor5 package.
+	 * @param {String} options.cwd Current working directory (packages) from which all paths will be resolved.
+	 * @param {String} options.packages Where to look for other packages (dependencies).
 	 * @returns {Promise}
 	 */
 	releaseDependencies( options ) {
+		const log = logger();
+
 		const execOptions = {
 			cwd: options.cwd,
 			packages: options.packages
 		};
 
-		const functionToExecute = ( repositoryName, repositoryPath ) => {
+		const errors = [];
+
+		return packagesToRelease( execOptions )
+			.then( ( dependencies ) => {
+				options.dependencies = dependencies;
+
+				if ( dependencies.size === 0 ) {
+					log.warning( 'Not found packages to release.' );
```

Comment:
This error is so unclear that I had to analyse the code to understand why are you logging it.

When writing error messages, or any messages at all, think about the user, not about the implementation. The user does not know what "finding packages to release" means. Think about a more user oriented message. Perhaps something like:

None of the packages contains any changes since its last release. Aborting.